scripts/run_parsers.py

```diff
@@ -45,7 +50,19 @@ def get_version_for_tag(self, tag) -> Tuple[int, int]:
           # try to read <vehicle>/version.h
           path1 = Path(f'{self.repository_path}/{tag["matches"]["name"]}/version.h')
           path2 = Path(f'{self.repository_path}/{self.valid_name_map[tag["matches"]["name"]]}/version.h')
-          file = path1 if path1.exists() else path2
+          path3 = Path(f'{self.repository_path}/Tools/{tag["matches"]["name"]}/version.h')
+          path4 = Path(f'{self.repository_path}/Tools/{self.valid_name_map[tag["matches"]["name"]]}/version.h')
+          if path1.exists():
+            file = path1
+          elif path2.exists():
+            file = path2
+          elif path3.exists():
+            file = path3
+          elif path4.exists():
+            file = path4
+          else:
+            raise FileNotFoundError(f'No version.h found for tag {tag["matches"]["name"]}')
```

Comment:
```suggestion
          potential_paths = [
            Path(f'{self.repository_path}/{tag["matches"]["name"]}/version.h'),
            Path(f'{self.repository_path}/{self.valid_name_map[tag["matches"]["name"]]}/version.h'),
            Path(f'{self.repository_path}/Tools/{tag["matches"]["name"]}/version.h'),
            Path(f'{self.repository_path}/Tools/{self.valid_name_map[tag["matches"]["name"]]}/version.h'),
          ]
          if (file := next((path for path in potential_paths if path.exists()), None)) is None:
            raise FileNotFoundError(f'No version.h found for tag {tag["matches"]["name"]}')
```
This isn't particularly necessary, but it does make it easier to add additional options in future...